src/containers/SearchPage/components/form/SearchConceptForm.tsx

```diff
@@ -39,6 +41,21 @@ const SearchConceptForm = ({ search: doSearch, searchObject: search, subjects }:
     },
   );
 
+  const { data: responsibles } = useAuth0Responsibles(
+    { permission: DRAFT_WRITE_SCOPE },
```

Comment:
Her kunne vi kanskje brukt concept:write for det er kanskje meir naturlig i og med at det er forklaringer det er snakk om.